### DISCUSSION TITLE
cpanel [Aug 16]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#3308] [Discord Threads]
> 🙋‍♂️ **MightyPannkaka**  
## Can someone explain to me how to repair this? This shows whenever i run the npm start  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/1006482891120844881/unknown.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### if u've experience with running node in the cpanel  
#  
> 💁 **o1lab**  
#### if you've experience running node with cpanel  
#  
> 💁 **o1lab**  
#### it should be similar  
#  
> 🙋‍♂️ **MightyPannkaka**  
#### Thank you for taking your time to response to my concern. Though this is my first time using node and npm in cpanel and localhost, as i follow the instruction and doesn't have an idea to fix some unexpected errors.  
#  
> 🙋‍♂️ **MightyPannkaka**  
#### Hello <@310157751395287040>, Can you guide me how to use the .env settings for cpanel? I use this settings on my .env and i still can't access the dashboard of the nocodb.   
####    
#### #PORT=8081         
#### #NC_DB=mysql2://localhost:3306?u=root&p=root&d=root_db   
#### #NC_PUBLIC_URL=https://mywebsite.com  
#  
> 💁 **o1lab**  
#### also pls try setting NC_BACKEND_URL  
#  
> 💁 **o1lab**  
#### which service are you using  
#  
> 💁 **o1lab**  
#### we've not used cpanel in deploying so hard to answer this   
#### however, i'll try to use the service you are trying  
#  
> 🙋‍♂️ **MightyPannkaka**  
#### Yes, I add the    
#### PORT   
#### NC_BACKEND_URL   
#### NC_PUBLIC_URL   
#### NC_DB  
#  
> 👨‍💻 **pranavxc**  
#### https://docs.cpanel.net/knowledge-base/web-services/how-to-install-a-node.js-application/  
#  
> 👨‍💻 **pranavxc**  
#### https://blog.cpanel.com/how-to-host-a-node-js-application-with-cpanel/  
#  
> 🙋‍♂️ **MightyPannkaka**  
#### Hello Mr. Pranavxc what should i do next after the nodejs application installation?  
#  
> 🙋‍♂️ **MightyPannkaka**  
#### And i move the nocodb-seed files inside the nodejs application folder. Can you guide me.  
![image0](https://cdn.discordapp.com/attachments/1006482891565432834/1008599767045128212/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/1006482891565432834/1008599767397441556/unknown.png)  
![image2](https://cdn.discordapp.com/attachments/1006482891565432834/1008599767741370368/unknown.png)  
#  
> 🙋‍♂️ **MightyPannkaka**  
#### and when i try to change the application root file in the nodejs. it shows the message  
![image0](https://cdn.discordapp.com/attachments/1006482891565432834/1008603984073850900/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/1006482891565432834/1008603984392638585/unknown.png)  
#  
> 💁 **o1lab**  
#### Hey, looks like we helped u to get running last time and asked to leave comments so that others could benefit next.  
#  
> 🙋‍♂️ **MightyPannkaka**  
#### Hello, Sorry for not leaving a comment due to i haven't seen the UI dashboard since last time.  
#  
> 🙋‍♂️ **MightyPannkaka**  
#### The last i receive was to send the cpanel support to help about the port.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1006482891565432834)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
